pegasus/sites.v3/hourofcode.com/public/css/front-page.css

```diff
@@ -48,8 +48,20 @@ a.category-link {
   opacity: 0.7;
 }
 
-.robotics-logo {
-  margin-right: 10px;
+.robotics-providers {
+  display: flex;
+  justify-content: space-between;
+  flex-wrap: wrap
+}
+
+.robotics-provider {
+  width: 182px;
+  height: 210px;
+  xfloat: left;
```

Comment:
typo or is "xfloat" something I don't know about?

Reply:
Oops, good catch.  It was a cheap way to "comment out" the line.